docs/Catalog/Preview.md

```diff
@@ -84,6 +84,14 @@ the iframe gets the following origin:
 > An important implication of same-origin policy is that the scripts
 > executed under the same origin share LocalStorage data and cookies.
 
+### Cross-Origin Resource Sharing (CORS)
+
+> New in Quilt Platform version 1.59.0 or higher
+
+Enabling permissive rendering also disables CORS protections, so that forms and
```

Comment:
it does not disable protections, it allows cross-origin resource sharing (meaning the scripts in html files can use external CORS-aware APIs). also, this was not changed in this release -- this was the case from the initial PFS implementation.
allowing forms and popups is new in this release, but it's kinda orthogonal to CORS.
also, i don't see any difference in how safari handles this.
i think you should just drop this section, it's misleading.

Reply:
I do think it worth documentating "forms and popups" only work on newer releases.

I will send you my credentials so you can test on Safari.